addons/survey/data/mail_template_data.xml

```diff
@@ -4,7 +4,7 @@
         <record id="mail_template_user_input_invite" model="mail.template">
             <field name="name">Survey: Invite</field>
             <field name="model_id" ref="model_survey_user_input" />
-            <field name="subject">Participate to {{ object.survey_id.display_name }} survey</field>
+            <field name="subject">Participate to {{ ctx.get('survey_display_name') or object.survey_id.display_name }} survey</field>
```

Comment:
I don't understand all this deal with the subject, the context usage, and the "render_field".

Can't we instead remove the subject in the xml data and make the compute method in the model hardcode the subject?
This seems a bit complex.

We could even keep it as is here if the mail.template used in another place, but still avoid all the render_field stuff and hardcode the subject in the model anyway.

Maybe I missed the point.

Reply:
Well, the purpose is to keep a well formed template you can use freely in chatter (with a subject), but allow a custom inject from the wizard. Effectively we could also force the subject in the wizard and discard the one from template, but then if somehow you use another template, the subject will always be forced instead of taking your new custom template's subject.

Concerning render_field it is used to generate a content based on a field containing qweb or inline qweb. Otherwise currently you have the raw content as it is considered as being a "mass mail" mode. In mass mode, we display raw content and generate at sending time. Bbut actually it will always be used on user inputs coming from the same survey, aka the subject should always be the same (which is not true if you configure another subject like "Invite for {{object.name}} to participate to {{object.survey_id.name}}", hence invalidating my first point (mmmh)).